### DISCUSSION TITLE
headingDivider as local directive?

### DISCUSSION THREAD

**Discussion by Chaphasilor** [marp-team/marp#64] [Q&A] [answered]
Is there a way to use the `headingDivider` as a local directive?  
Right now it seems like it is always global...

I don't want the `headingDivider` to apply to the first slide, only to the other ones :)

**Comment by yhatt** [comment, marked as answer]
Unfortunately `headingDivider` is [the global directive](https://marpit.marp.app/directives?id=global-directives-1) so cannot turn on in the middle of slides.

Marpit framework is parsing local directives after determined the point for splitting slides. `headingDivider` as the local directive becomes difficult to determine applicable pages because may change split points for pages dynamically while parsing.

Having said that, I think it's a fair demand. It could be accepted as a request issue in [the issue tracker of Marpit framework](https://github.com/marp-team/marpit/issues).

**Comment by stevenwdv** [comment]
If you can use HTML, you can use e.g. `<h1>`/`<h2>`/... instead of markdown headings, as they never trigger a new slide.
